### ISSUE TITLE
Coordinating the official integration into QtCreator 

### ISSUE THREAD

**Issue by ShalokShalom** [sassanh/qnvim#11]
The original author of this software, @sassanh is currently a little busy. 
He has asked us for help, to get his project upstream: https://github.com/sassanh/qnvim/issues/1#issuecomment-529593191

One issue is the license, as brought up by someone in #qt on Freenode. 

This issue here is meant to lead through this process.

**Comment by Shatur95** [comment]
Until this is not merged into QtCreator, I made an AUR package: [qnvim-git](https://aur.archlinux.org/packages/qnvim-git). I will provide installation instructions today after work.

**Comment by sassanh** [comment]
> Until this is not merged into QtCreator, I made an AUR package: [qnvim-git](https://aur.archlinux.org/packages/qnvim-git). I will provide installation instructions today after work.

I'm glad qnvim s more reachable now thanks to your help.

An update about state of this integration:
I had an email from Tino Pyssysalo, product owner of Qt Creator and Qt Marketplace, Tino suggested that considering it depends on neovim-qt, it will be difficult to embed this plugin inside Qt Creator, so he suggested to use Qt Marketplace for this purpose.
I think it makes sense, so we better present it in marketplace, meanwhile having it in distros like what @Shatur95 did for Arch would be helpful for users.

**Comment by Shatur95** [comment]
I think the plugin requires some polishing before publication. I will try to help you with it.

**Comment by sassanh** [comment]
Yes it needs polishing and unfortunately I'm out of time, your help is appreciated.

**Comment by ShalokShalom** [comment]
I can pack it for KaOS
Just like to see it in an usable state.
Thanks a lot for all your effort, appreciate it

**Comment by bam80** [comment]
Any progress?
